src/css/override_vars.scss

```diff
@@ -10,6 +10,7 @@ $color-red:                     #be4900;
 
 $color-trueblack:               #000000;
 $color-textblack:               #333333;
+$color-textlighter:             #525252;
```

Comment:
This seems a bit funny to me. Why are we adding a new color? I think I would prefer just to do one of the color [transform functions](http://sass-lang.com/documentation/Sass/Script/Functions.html#lighten-instance_method) to lighten one of the existing colors.